exampleCourse/questions/demo/workspace/vscode/info.json

```diff
@@ -11,6 +11,7 @@
         "image": "codercom/code-server",
         "port": 8080,
         "args": "--auth none",
+        "home": "/home/coder/project",
```

Comment:
I think in this example we should actually make this `/home/coder`, so that it's the real home directory?

Reply:
Hmm should we just rename the option then? Because I think we still need the `.../project` part to be configurable?

Reply:
Why do we need that to be configurable? Can't you just make the `project` directory inside `questions/XXX/workspace/project` if you want it?

Reply:
Ah ok